### ISSUE TITLE
`cuerdas.impl.xregexp` namespace not available in shadow-cljs projects

### ISSUE THREAD

**Issue by rschmukler** [funcool/cuerdas#81]
I'm currently trying to get version `2021.05.02-0` of cuerdas working with a shadow-cljs project. I've added it to my `deps.edn` file (and have shadow configured to use it) but upon trying to require `cuerdas.core` I get the following error:

```
The required namespace "cuerdas.impl.xregexp" is not available, it was required by "cuerdas/regexp.cljc".
```

Interestingly, version `2020.03.26-3` (which, per the changelog, doesn't include the shadow-cljs compatibility fix) does work with shadow version `2.12.5` (latest on clojars as of 4/13/21).

**Comment by niwinz** [comment]
Bumped the `2021.05.09-0`, please can you check if it works out of the box on your shadow-cljs setup?

**Comment by rschmukler** [comment]
Confirmed as fixed in `2021.05.09-0` - thank you so much for the quick fix and all of your open source work! Cheers!
